core/server/master/src/main/java/alluxio/master/metrics/MetricsStore.java

```diff
@@ -224,7 +224,8 @@ public void initMetricKeys() {
             long total =
                 MetricsSystem.counter(MetricKey.CLUSTER_BYTES_READ_DIRECT.getName()).getCount()
                 + MetricsSystem.counter(MetricKey.CLUSTER_BYTES_READ_REMOTE.getName()).getCount()
-                + MetricsSystem.counter(MetricKey.CLUSTER_BYTES_READ_DOMAIN.getName()).getCount();
+                + MetricsSystem.counter(MetricKey.CLUSTER_BYTES_READ_DOMAIN.getName()).getCount()
+                + MetricsSystem.counter(MetricKey.CLUSTER_BYTES_READ_LOCAL.getName()).getCount();
```

Comment:
this change is unrelated and already merged in by da3210a470025172dc9d49ea246909ca61305549, right?

Reply:
Yes, sorry, I merged local master and add this change by accident, should I do something for this?